src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml

```diff
@@ -89,216 +89,225 @@
         </Style>
```

Comment:
Please see the `References` section in the main description to understand how parts of this is restructured (it is impossible to read these differences in `XAML` file).